eZ/Bundle/EzPublishCoreBundle/Command/CleanupVersionsCommand.php

```diff
@@ -133,6 +154,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $contentIdsCount
         ));
 
+        $displayProgressBar = !($output->isVerbose() || $output->isVeryVerbose() || $output->isDebug());
```

Comment:
Well, I thought the same but once I checked, it turn-out to behave slightly different. `ProgressBar` is displayed for all verbosity settings excluding `OutputInterface::VERBOSITY_QUIET` only, what is not expected in this command. 
I want to display `ProgressBar` for `OutputInterface::VERBOSITY_NORMAL` only and for `OutputInterface::VERBOSITY_VERBOSE` or "higher", I want to display more detailed information. The following screenshots shows how it behaves without using `$displayProgressBar`, take a look please:

<img width="747" alt="Screenshot 2019-07-08 at 18 32 00" src="https://user-images.githubusercontent.com/10659510/60826942-4f1a8e80-a1af-11e9-9c96-3a4a5784af61.png">

Unfortunately, even if I really want, I couldn't keep both `ProgressBar` and detailed info when verbose is used, because the output is changing so fast and `ProgressBar` is not visible at all (if I follow the  Symfony's docs advice about using `clear()` and `display()` `ProgressBar` methods 😉) or is written to the output after each change, what doesn't look good (as you can see on the screenshot).

Reply:
> `ProgressBar` is displayed for all verbosity settings excluding `OutputInterface::VERBOSITY_QUIET` only, what is not good enough for me. 

Hmm, yeah, this is usually the expected way, but maybe not for all the cases then. Thanks for the entire clarification.